### PR TITLE
Fixing errors on trivia questions

### PR DESCRIPTION
**Pull request by jphjsoares**
Fixing ONLY the errors mentioned on https://github.com/GilbertGobbels/GAwesomeBot/issues/187.

### CODE REVIEW

**Comment by vladfrangu** [comment]
So. Hi. I'm leaving you this message for multiple reasons.

## Thanks for submitting a PR.

We appreciate all pull requests we might get, as they allow multiple people to contribute.

## This PR is going to the development branch

Which is probably why it won't get merged 👀 
We're currently working very hard on the rewrite branch, you know `indev-4.0.2`. As such, try pointing your Pull Request there. There's still a lot of work to do, and Trivia is not one of our priorities. But, just like everything, it'll get fixed.

## TL;DR;

Try pointing your branch to `indev-4.0.2`. That will definitely increase the chance for your PR to merge (hopefully)

**Comment by Gilbert142** [comment]
Please clone the indev-4.0.2 branch, copy your edits to the trivia.json there and open a new PR pointing to our indev-4.0.2 👍
